modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-channel-impl/src/main/java/com/liferay/headless/commerce/admin/channel/internal/resource/v1_0/ProductDisplayPageResourceImpl.java

```diff
@@ -169,30 +174,31 @@ private Map<String, Map<String, String>> _getActions(
 
 		return HashMapBuilder.<String, Map<String, String>>put(
 			"delete",
-			addAction(
-				ActionKeys.DELETE, cpDisplayLayout.getCPDisplayLayoutId(),
+			ActionUtil.addAction(
+				ActionKeys.DELETE, getClass(), cpDisplayLayout.getClassPK(),
```

Comment:
this I still believe should be the UPDATE, because in the end you will not check the DELETE permission but the UPDATE permission on the product to delete this page

Reply:
@alessiorendina can you please have a look at this

Reply:
Hey @andrea-ale-sbarra, I think you are correct about this. Looking at the permission check in `CPDisplayLayoutServiceImpl` for `deleteCPDisplayLayout`,  `ActionKeys.UPDATE` is being checked. I'll make the correction.